### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by OctopusET** [OctopusET/sway-contrib#9]
Should this follow sway's release or just make its own release cycles.

Release is mostly for the packaging and testing.

**Comment by b1rger** [comment]
Hi,

whether you're following sways release or make your own release cycles - it would be great if you would start with a release greater than 1.8.1. We packaged grimshot in Debian with that version number - and [as far I can see we are not the only ones](https://pkgs.org/search/?q=grimshot)

**Comment by OctopusET** [comment]
My idea is using [Calendar Versioning](https://calver.org). Or just wait for the sway 1.9 release, but I think it would be better to make its own cycles.
Or copy this https://github.com/hyprwm/contrib/releases, but it's lower than 1.8.1.

Would there be a problem with Debian packaging if I use Calver?

**Comment by b1rger** [comment]
> Would there be a problem with Debian packaging if I use Calver?

no, not at all, there is a lot of software that uses Calver packaged in Debian. If the version starts with `24` or `2024` that will be fine, because its greater than `1`

**Comment by fossdd** [comment]
Hi, is there a new release planned as sway 1.10 is released?

**Comment by OctopusET** [comment]
@fossdd Yeah, I was busy for a while. I will try release it in this week.
